CONTRIBUTORS.md

```diff
@@ -0,0 +1,20 @@
+# Contributors
+
+The ODL maintainers are grateful to acknowledge contributions from several people. What follows is an incomplete list, see [contributors](https://github.com/odlgroup/odl/graphs/contributors) for a complete list and feel free to add yourself!
+
+## Contributors in alphabetic order
+
+* [Jonas Adler](https://github.com/adler-j). Package maintainer. Created much of the `Set`, `LinearSpace` and `Operator` structure including utilities. Has also contributed to most other submodules.
+* [Chong Chen](https://github.com/chongchenmath). Started work on the `odl.deform` package.
+* [Matthias J. Ehrhardt](https://github.com/matthiasje). Several contributions to `odl.solvers`, in addition to general bug fixes and improvements.
+* [Barbara Gris](https://github.com/bgris). Added `examples/solvers/find_optimal_parameters`.
+* [Johan Karlsson](https://github.com/hilding79). Contributions to `odl.solvers`.
+* [Holger Kohr](https://github.com/adler-j). Package maintainer. Was part of the design of ODL and created several of the submodules, including the `odl.discr` and `odl.trafos` modules. Has contributed to most modules.
```

Comment:
I existed before ODL, so I wasn't part of the design 😛 
 
"including \`odl.discr\` and \`odl.trafos\`"  otherwise repetitive "modules"

Reply:
I also want 3 things :-), so please add the tomographic geometries stuff

Reply:
Will do!